### PR TITLE
RM-7408: Extract Remotion.Development.Core.UnitTesting.Sandboxing 

### PR DESCRIPTION
**Pull request by dkudernatsch**
https://re-motion.atlassian.net/browse/RM-7408

### CODE REVIEW

File: Remotion/Development/Sandboxing.NUnit2/Development.Sandboxing.NUnit2.csproj [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    <AssemblyOriginatorKeyFile>..\..\..\remotion.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="nunit.framework, Version=2.6.3.13283, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77">
+      <HintPath>..\..\..\packages\NUnit.2.6.3\lib\nunit.framework.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Drawing" />
```
Comment (MichaelKetting) [inline Remotion/Development/Sandboxing.NUnit2/Development.Sandboxing.NUnit2.csproj:48]: Do we need Core, Drawing, and Data?

File: Remotion/Development/Sandboxing.NUnit2/Development.Sandboxing.NUnit2.csproj [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE;JETBRAINS_ANNOTATIONS</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```
Comment (MichaelKetting) [inline Remotion/Development/Sandboxing.NUnit2/Development.Sandboxing.NUnit2.csproj:24]: those settings aren't copied from the correct sample, e.g. a production code project.

Generate-Docs and NoWarn entry is missing in release and debug.

File: Remotion/Development/Sandboxing.NUnit2/Development.Sandboxing.NUnit2.csproj (hunk 1 above)
Comment (dkudernatsch) [reply Remotion/Development/Sandboxing.NUnit2/Development.Sandboxing.NUnit2.csproj:51]: Data and Drawing are both used as part of the PermissionSets for partial trust
Core appears to be unused and can be removed.

**Comment by Stefan-Ilic** [comment]
Added to `master` with 03b85116cff8629536e27e9a531cef5d444ea4ba

**Pull request closed without merging**